src/features/surveys/rpc/getSurveyStats.ts

```diff
@@ -39,7 +40,7 @@ async function handle(
   );
 
   let sum = 0;
-
+  let unlinkedCount = 0;
```

Comment:
You took away the whitespace here. I don't like it. 😉

Reply:
Got it! 👍